snowplow/iglu-client-embedded/schemas/com.metabase/dashboard/jsonschema/1-1-0

```diff
@@ -1,47 +1,63 @@
 {
   "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
-  "description": "Dashboard tab events",
+  "description": "Dashboard events",
```

Comment:
This is the copy of the previous `dashboard` schema (version `1-0-1`) with new properties from `dashboardtab` with `null` types.